manager-bundle/skeleton/public/.htaccess

```diff
@@ -10,6 +10,155 @@
+                                                  "text/vcard" \
+                                                  "text/vnd.rim.location.xloc" \
+                                                  "text/vtt" \
+                                                  "text/x-component" \
+                                                  "text/x-cross-domain-policy" \
+                                                  "text/xml"
+        </IfModule>
+
+        # Gzip compression
+        <IfModule mod_deflate.c>
+            AddOutputFilterByType DEFLATE "application/atom+xml" \
+                                          "application/javascript" \
+                                          "application/json" \
+                                          "application/ld+json" \
+                                          "application/manifest+json" \
+                                          "application/rdf+xml" \
+                                          "application/rss+xml" \
+                                          "application/schema+json" \
+                                          "application/geo+json" \
+                                          "application/vnd.ms-fontobject" \
+                                          "application/wasm" \
+                                          "application/x-font-ttf" \
+                                          "application/x-javascript" \
+                                          "application/x-web-app-manifest+json" \
+                                          "application/xhtml+xml" \
+                                          "application/xml" \
+                                          "font/eot" \
+                                          "font/opentype" \
+                                          "font/otf" \
+                                          "font/ttf" \
+                                          "font/woff2" \
+                                          "image/bmp" \
+                                          "image/svg+xml" \
+                                          "image/vnd.microsoft.icon" \
+                                          "image/x-icon" \
+                                          "text/cache-manifest" \
+                                          "text/calendar" \
+                                          "text/css" \
+                                          "text/html" \
+                                          "text/javascript" \
+                                          "text/plain" \
+                                          "text/markdown" \
+                                          "text/vcard" \
+                                          "text/vnd.rim.location.xloc" \
+                                          "text/vtt" \
+                                          "text/x-component" \
+                                          "text/x-cross-domain-policy" \
+                                          "text/xml"
+        </IfModule>
```

Comment:
All of these are within the `<IfModule mod_rewrite.c>` block - but they are not related to `mod_rewrite`. They should be outside, shouldn't they?

Reply:
Good point, just changed that.